lib/Comment.php

```diff
@@ -31,7 +31,7 @@
  * <p class="comment-attribution">- Sullivan Ballou</p>
  * ```
  */
-class Comment extends Core implements CoreInterface {
+class Comment extends Core implements CoreInterface, MetaInterface {
```

Comment:
What do we want to do with that CoreInterface in the future? It only has a very limited functionality right now.

Reply:
I’ve thought about this, too. See this discussion: https://github.com/timber/timber/issues/478#issuecomment-442256153.